pom.xml

```diff
@@ -1,113 +1,100 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <modelVersion>4.0.0</modelVersion>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
 
-    <parent>
-        <groupId>de.samply</groupId>
-        <artifactId>parent</artifactId>
-        <version>11.1.0</version>
-    </parent>
+  <parent>
+    <groupId>de.samply</groupId>
+    <artifactId>parent</artifactId>
+    <version>11.1.0</version>
+  </parent>
 
-    <artifactId>common-config</artifactId>
-    <version>5.1.0-SNAPSHOT</version>
-    <packaging>jar</packaging>
+  <artifactId>common-config</artifactId>
+  <version>5.1.0</version>
```

Comment:
Please change to 5.1.1-SNAPSHOT. 5.1.0 seems to be the currently released version.